### PR TITLE
Add com console parameters for esxi

### PR DESCRIPTION
**Pull request by nortonluo**
In many rack mount servers or customized servers, there is no monitor connected, UART port are usually used as console. VMware ESXi use VGA card as console. User need pass VM kernel To support this use, I add three options: "gdbPort","logPort","debugLogToSerial".
VMware ESXi reference

Refer to code change in https://github.com/RackHD/on-tasks/pull/443

### CODE REVIEW

**Comment by nortonluo** [comment]
@anhou @iceiilin

File: docs/rackhd/install_os.rst [hunk 1]
```diff
@@ -249,8 +249,11 @@ installPartitions   Array            *optional*   *installPartitions* is to spec
...
 postInstallCommands Array            *optional*   **(ESXi only)** If specified, this contains an array of string commands that will be run at the end of the post installation step.  This can be used by the customer to tweak final system configuration.
-installType         String           *optional*   **(PhotonOS only)** The value is *minimal* or *full* to indicate the type of installed OS, defualt *installType* is **minimal**
-installScriptUri    String           *optional*   The download URI for a custom kickstart/preseed/autoyast/cloud-config template to be used for automatic installation/configuration.
+installType         String           *optional*   **(PhotonOS only)** The value is *minimal* or *full* to indicate the type of installed OS, defualt *installType* is **minimal**
+gdbPort             String           *optional*   **(ESXi only)** Serial port used for GDB debug
+logPort             String           *optional*   **(ESXi only)** Serial port for log output in ESXi
```
Comment (pengz1) [inline docs/rackhd/install_os.rst]: Please add a valid example here. like "com0", "com1", or a specified link as how to set gdbPort and logPort.

**Comment by anhou** [comment]
@nortonluo I think it's not a good way to add so many OS-specific field, it's better to summary the same type filed for several OSes, just an example like, logEnabled: true, which could be across Ubuntu/CentOS/ESXi, if the behavior is different, we could annotate them in doc. another benifit is users don't need to care about how to input comX or serialX. just tell user/developer where is the log.

**Comment by lanchongyizu** [comment]
@nortonluo could you resolve the conflict?

**Pull request closed without merging**